### ISSUE TITLE
Unable to search the Help modal

### ISSUE THREAD

**Issue by andreincu** [excalidraw/excalidraw#8534]
**Problem**
I noticed that after implementing the "Find on Canvas" feature, it has become impossible to search for a specific keyboard shortcut in the "Help" modal using the browser's "Find in Page" shortcut. 

This issue seems to have arisen because the event appears to be intercepted by the custom implementation.

**Expected**
"Find on canvas" should be triggered only while you're focusing the canvas, right?

**Comment by imran1khan** [comment]
@andreincu i  thing the easy way to solve this is that we should chnage the shortcut for "Find on Canvas", to something like 
[ ctrl+shift+f ], what do you thing?

**Comment by andreincu** [comment]
Yep, that's also a viable solution and simpler, compared to triggering the action based on what element in on focus. 😄

**Comment by imran1khan** [comment]
@andreincu hay bud, please currect me if i am wrong
 are you asking for a search feature, like select the text area and nly search on that paticular text area? just asking

**Comment by imran1khan** [comment]
hi @Mrazator , if you want i can work on this
